dashboard/state_aggregator.py

```diff
@@ -35,110 +33,95 @@ class StateAPIManager:
 
     def __init__(self, state_data_source_client: StateDataSourceClient):
         self._client = state_data_source_client
-        DataSource.nodes.signal.append(self._update_raylet_stubs)
```

Comment:
I moved it to `state_head.py` which is a lot nicer because the StateAPIManager can be independent from dashboard in this way